build/workbox.config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "globDirectory": "./",
+    "globPatterns": [
+        "assets/**/*.{js,svg,png,css}",
+        "dist/**/*.{js,css}",
+        "docs/**/*.{md,html,css,jpg,js}"
```

Comment:
Okay so I added a [transformer](https://github.com/twbs/bootstrap/pull/23533/files#diff-7382067a8603719f3dfffe1afa96738dR7) for this as if I specified the _gh_pages folder, workbox will attempt to pre-cache an url like.. http://getbootstrap.com/gh_pages/docs/4.0/**, instead of http://getbootstrap.com/docs/4.0/**. The transformer basically renames the .md files to their correct .html files but keeps the docs/ structure intact instead of _gh_pages/docs/